pom.xml

```diff
@@ -148,48 +151,33 @@
                 <include>package-lock.json</include>
               </includes>
             </fileset>
+<!--            <fileset>-->
+<!--              <directory>node</directory>-->
+<!--              <includes>-->
+<!--                <include>**/*</include>-->
+<!--              </includes>-->
+<!--            </fileset>-->
+<!--            <fileset>-->
+<!--              <directory>node_modules</directory>-->
+<!--              <includes>-->
+<!--                <include>**/*</include>-->
+<!--              </includes>-->
+<!--            </fileset>-->
             <fileset>
-              <directory>node</directory>
-              <includes>
-                <include>**/*</include>
-              </includes>
-            </fileset>
-            <fileset>
-              <directory>node_modules</directory>
+              <directory>${project.basedir}/src/main/webapp</directory>
               <includes>
                 <include>**/*</include>
               </includes>
+              <excludes>
+                <exclude>css/custom-prism.css</exclude>
+              </excludes>
+              <excludes>
+                <exclude>js/theme-loader.js</exclude>
```

Comment:
might be better to create a prism folder and one for this plugin so we don't need this ignore rules?

Reply:
This makes somehow sense? Or does that not work with the hpi plugin?